Lib/fontTools/pens/ttGlyphPen.py

```diff
@@ -47,36 +55,127 @@ def __init__(self, glyphSet, handleOverflowingTransforms=True):
         self.handleOverflowingTransforms = handleOverflowingTransforms
         self.init()
 
-    def init(self):
+    def init(self) -> None:
         self.points = []
         self.endPts = []
         self.types = []
         self.components = []
+        self.currentPath = None
```

Comment:
self.currentPath is not used by TTGlyphBasePen nor by TTGlyphPen, but only by TTGlyphPointPen. It would make more sense to have it defined only in the latter's `__init__` method, after the `super().__init__()` has been called.

Reply:
As TTGlyphPointPen has no `__init__` method yet, I think leaving an unused instance variable probably doesn't hurt, WDYT?